components/registry-facade/cmd/run.go

```diff
@@ -82,6 +85,7 @@ var runCmd = &cobra.Command{
 		if err != nil {
 			log.WithError(err).Fatal("cannot create registry")
 		}
+		go watchConfig(args[0], reg)
```

Comment:
I'd prefer something like `var configPath = args[0]` at the very top of this function instead of referencing `args[0]` twice